src/video/src/Video_PSP.cpp

```diff
@@ -2,7 +2,7 @@
 // Created by dbeef on 7/7/19.
 //
 #include "video/Video.hpp"
-#include "glad/glad.h"
+#include "SDL_opengl.h"
```

Comment:
All `glad/glad.h` replacements need to be opaque. Possibly add a dummy `glad` target for emscripten.